.github/scripts/merge.py

```diff
@@ -0,0 +1,62 @@
+import os
+import errno
+
+exts = ["js", "ts"]
+scenarios = ["default"]
+me_file_name = "messageExtensionBot"
+bot_file_name = "teamsBot"
+combined_file_name = "./bot-msgext/teamsBot"
+for ext in exts:
+    for scenario in scenarios:
+        me_file = "./templates/msgext/{}/{}/{}.{}".format(ext, scenario, me_file_name, ext)
+        bot_file = "./templates/bot/{}/{}/{}.{}".format(ext, scenario, bot_file_name, ext)
+        combined_file = "./templates/bot-msgext/{}/{}/{}.{}".format(ext, scenario, bot_file_name, ext)
+        if not os.path.exists(os.path.dirname(combined_file)):
+            try:
+                os.makedirs(os.path.dirname(combined_file))
+            except OSError as exc:  # Guard against race condition
+                if exc.errno != errno.EEXIST:
+                    raise
+                
+        bot_fh = open(bot_file)
+        bot_content = bot_fh.readlines()
+        me_fh = open(me_file)
+        me_content = me_fh.readlines()
+
+        # merge pkg reference
+        for line in me_content:
+            if not line.startswith(("import", "const")):
+                break
+            if "botbuilder" not in line and line not in bot_content:
```

Comment:
How do we handle this line in ME file? skip or merge the content of "TeamsActivityHandler, CardFactory" to Bot file?
```
const { TeamsActivityHandler, CardFactory } = require("botbuilder");
```

Reply:
Yes Bot's pkg include ME's. Use Bot's statement is ok